samples/mediator-ws.ts

```diff
@@ -49,8 +49,16 @@ class WsInboundTransporter implements InboundTransporter {
   }
 }
 
+// TODO: use WsOutboundTransporter from the agent
 class WsOutboundTransporter implements OutboundTransporter {
-  public supportedSchemes = ['ws']
+  public supportedSchemes = ['ws', 'wss']
```

Comment:
I haven't tested `wss` and I don't know much details about that protocol so I'm not sure if it will work out of the box.

Reply:
Both `ws` and the React Native WebSocket client supports `wss`.